features/step_definitions/networking.rb

```diff
@@ -450,8 +450,9 @@
 
 Given /^I wait for the networking components of the node to be terminated$/ do
   ensure_admin_tagged
+  # TODO: this doesn't mean terminated, it means killed, but the pods restart
 
-  if env.version_ge("3.10", user: user)
+  unless env.version_ge("4.6", user: user)
```

Comment:
And what if version is >= 4.6? Shall we fail?

Reply:
replacing this with `systemctl is-enabled openvswitch` and dropping 3.x support
```
  _host = node.host
  @result = _host.exec_admin("systemctl is-enabled openvswitch")
  if @result[:success] && (@result[:response].include? "enabled")
``